<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QPushButton" name="button_openmapmaker">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY견고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>맵 생성 및 확인하기 </string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_loadMap">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY견고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>맵 불러오기 </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY견고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>로봇 정하기 </string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_paramset">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY견고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>시물레이션 파라미터 
 설정 </string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_simStart">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY견고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>정적 시물레이션</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_simStart_dynamic">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY견고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>동적 시물레이션</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
